app/jobs/course/assessment/answer/auto_grading_job.rb

```diff
@@ -41,17 +43,30 @@ class Course::Assessment::Answer::AutoGradingJob < ApplicationJob
   # @param [String] redirect_to_path The path to redirect when job finishes.
   def perform_tracked(answer, redirect_to_path = nil)
     ActsAsTenant.without_tenant do
+      raise PriorityShouldBeLoweredError if queue_name == 'highest' && answer.question.is_low_priority
```

Comment:
nit: consider checking that 'delayed' not in the `queue_name` instead, then an identical check can be used in both job files.

Reply:
done